Creating-Objects.md

```diff
@@ -31,7 +31,7 @@ The schema would match a dictionary like this:
 {"version": 34}
 ```
 
-The `dict` type is a collection, but the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
+This schema describes a dictionary of ints, and the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
```

Comment:
Doesn't this describe a dict of one key-value pair where the key is 'version' and the value is an int? This has a different sense than "a dictionary of ints".

Reply:
yep! fixed